plugins/chat/spec/system/page_objects/chat/components/message.rb

```diff
@@ -32,14 +32,56 @@ def expand
 
         def secondary_action(action)
           if page.has_css?("html.mobile-view", wait: 0)
-            component.find(".chat-message-text").click(delay: 0.6)
+            open_mobile_actions
             page.find(".chat-message-actions [data-id=\"#{action}\"]").click
           else
             open_more_menu
             page.find("[data-value='#{action}']").click
           end
         end
 
+        def open_mobile_actions
+          page.execute_script(<<-JS, component, 0.6)
```

Comment:
will remove this 0.6 and arguments[1] * 1000, but don't want to stop tests for this